crate_universe/private/crates_vendor.bzl

```diff
@@ -41,7 +41,7 @@ def _is_windows(ctx):
 
 def _get_output_package(ctx):
```

Comment:
I think the original intent here was that users could set `/path/to/something` and get a relative path from the workspace root. Would it not also work call `lstrip("/")` on `output` to continue support for both of these uses?